nimbus-ui/nimbusui/src/app/components/platform/validators/custom.validators.ts

```diff
@@ -73,6 +73,5 @@ export class CustomValidators {
             return null;
         };
     }
-
```

Comment:
In custom validator for @past and @future, we need to check if the value that calendar control has is actually less than today's date or greater than today's date respectively. The min and max date is just going to grey out the calendar UI from selecting those but the user can enter any date. 
Something more to consider - the @Past and @future constraints are specifically controlling the input based on today's date. If there is a use case to allow only a range of dates that would need more design and work.

Reply:
Agree on the first point.
> Something more to consider - the @past and @future constraints are specifically controlling the input based on today's date. If there is a use case to allow only a range of dates that would need more design and work.

It sounds like the whole point of this was to avoid using .drl files to validate date inputs. If this implementation solves that issue... then fine I say we consider date range later.